macos/FluentUI/Link/Link.swift

```diff
@@ -93,24 +97,29 @@ open class Link: NSButton {
 		self.trackingArea = trackingArea
 	}
 	
+	open override func mouseDown(with event: NSEvent) {
+		super.mouseDown(with: event)
+		updateTitle()
```

Comment:
I may be misunderstanding something, but why do we need a call to updateTitle here? We're not changing anything in this mouseDown implementation nor does the implementation of updateTitle seem to be using anything that would be affected by super.mouseDown?

Reply:
"Link with overridden Target/Action" (in the test app) disappears when clicked.  I wasn't able to figure out why that happened, but I can improve the behvior somewhat by fixing up the title as soon as the modal dialog is dismissed.  That's what this is for.

Reply:
What's the buttonType for this? Maybe needs to be .momentaryChange instead of the default?

Reply:
Behavior is unchanged if I `setButtonType(.momentaryPushIn)`, so that is probably the default.  `.momentaryChange` is a better fix: link doesn't disappear at all now.